esignet-core/src/main/java/io/mosip/esignet/core/dto/OAuthDetailRequest.java

```diff
@@ -76,6 +72,7 @@ public class OAuthDetailRequest {
      * names of the individual Claims being requested as the member names.
      */
     @Valid
+    @ClaimSchema(message = ErrorConstants.INVALID_CLAIMS_SCHEMA)
```

Comment:
As the query parameter is named as "claims" it is better to name as "ClaimsSchema". And also the error code can state as "INVALID_CLAIMS"

Reply:
changed